src/requirements/data/exams/ExamCredit.ts

```diff
@@ -1,235 +1,204 @@
 import { FWS_COURSE_ID } from '../constants';
 
-export type ExamRequirements = {
-  readonly name: string;
-  readonly fulfillment: {
-    readonly courseEquivalents?: Record<string, number[]>;
-    readonly minimumScore: number;
-    readonly credits: number;
-    readonly majorsExcluded?: string[];
-  };
+export type ExamFulfillment = {
+  readonly courseId: number;
+  readonly courseEquivalents?: Record<string, number[]>;
+  readonly minimumScore: number;
+  readonly credits: number;
+  readonly majorsExcluded?: string[];
 };
```

Comment:
Can you write a comment for what `ExamFulfillment` is? It's another one of those vague terms that feel very intuitive for us, but might be very confusing to a newbie